Updated limitations (#247)
mulesoft/docs-access-management: modules/ROOT/pages/configure-client-management-azure.adoc

For a list of supported grant types, see the https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow[Microsoft identity platform documentation].

== Limitations 

=== Microsoft National Clouds

> Anypoint Platform do not support Azure AD client providers deployed in Microsoft national clouds, e.g. Azure Goverment, Azure China 21Vianet. 

Reviewer: ```suggestion

Anypoint Platform does not support Azure AD client providers deployed in Microsoft national clouds, such as  Azure Goverment or Azure China 21Vianet. 
```
